### ISSUE TITLE
Implement settings for motion design

### ISSUE THREAD

**Issue by BrianSipple** [Ticketfly/ticketfly-ui#132]
@Ticketfly/frontenders @yrving 

I'd like to suggest implementing settings here related to motion design -- specifically animation duration and animation timing functions for different types of motion (think scaling, fading, sliding, bouncing, etc.)

I have Gists for each ([duration](https://gist.github.com/BrianSipple/e0cffe1e72ee573a1a11d8b0ed5b922a) and [easing](https://gist.github.com/BrianSipple/aef431b8d126aef6c7f60264c58dbf0e)) that I like to keep handy and have been tweaking for a few years now. They're written in the syntax of CSS variables, but if there's interest, I could certainly transcribe them to SCSS variables and add them in our `settings`.

The benefit would be that someone can define rules for an animation or transition -- for example, sliding out a modal -- and be able to reference a variable for duration/easing that would make the effect consistent with the Ticketfly Motion Design Language (which I just made up, but will totally be a thing 😛).

**Comment by bl4ckm0r3** [comment]
@BrianSipple yeah we should talk with @yrving and jake and allison because they are already working on the design guidelines and i think this should be part of it!
